scripts/assets/file_content_validation_checks.textproto

```diff
@@ -371,4 +373,25 @@ file_content_checks {
   prohibited_content_regex: "@color/(?!color_def_).+"
   failure_message: "Only colors from color_defs.xml may be used in color_palette.xml."
 }
-
+file_content_checks {
+  file_path_regex: ".+?ActivityTest.kt"
+  failure_message: "Screen Name test not added."
```

Comment:
For this & below, could you revise these failure messages to also provide some instruction for how to fix the issue? See other failure messages in this file for references.

Reply:
done.